crates/burn-tensor/src/tensor/api/float.rs

```diff
@@ -119,23 +137,34 @@ where
     /// }
     /// ```
     pub fn int(self) -> Tensor<B, D, Int> {
-        Tensor::new(B::float_into_int(self.primitive))
+        // NOTE: should `tensor.int()` return the quantized values?
```

Comment:
No since we don't support QInt.

Reply:
Whoops, that comment should have been removed 😄